File: corpus/cs3/0310-construindo-casas/solutions/a-otima-basica/main.py
import sys
import math

def main():
    input_data = sys.stdin.read().split()
    idx = 0
    while idx < len(input_data):
        a = int(input_data[idx])
        idx += 1
        if a == 0:
            break
        b = int(input_data[idx])
        c = int(input_data[idx+1])
        idx += 2
        
        area = a * b
        total = (area * 100) // c
        side = int(math.sqrt(total))
        print(side)

if __name__ == '__main__':
    main()

File: corpus/cs3/0310-construindo-casas/solutions/a-otima-basica/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        while (sc.hasNextLong()) {
            long a = sc.nextLong();
            if (a == 0) break;
            long b = sc.nextLong();
            long c = sc.nextLong();
            
            long area = a * b;
            long total = (area * 100) / c;
            long side = (long) Math.sqrt(total);
            System.out.println(side);
        }
        sc.close();
    }
}
